### PR TITLE
Задача 6. Компьютерное зрение: задача сегментации

### PR DESCRIPTION
**Pull request by artmotika**

### CODE REVIEW

**Comment by VladimirGl** [comment]
сложно графики читать, loss и jaccard в разных масштабах существуют, сравнивать нам их между собой смысла нету - так что лучше отдельно график где train и val jaccard и график где train и val loss

* добавили ауги - трансформ на картинку и маску применяется по отдельности - значит картинка может повернуться а маска нет => даем не правильный семпл в обучение (маска не соответствует картинке) => эт приводит к тому что лосс хуже чем без аугов :)
* поворот на до 180 (да и до 90) не оч имеет смысл для человеческих фоток ; небольшой да, а вверх ногами мы может и не ожидаем картинки (а может и ожидаем)
* ауги которые не релевантны тесту - могут ухудшать качество
* но любопытно что скор все равно адекватный 
* всегда имеет смысл глазами смотреть на то что dataloader вам отдает - эт оч частая ошибка чуть не верно обработать данные и искать че не так с архитектурой / пайплайоном :)

за пункт про ауги пока 1/2, остальное ок

**Comment by VladimirGl** [comment]
у вас и у https://github.com/IAmFunkyFrog/ml-course/pull/3 на удивление схожий мыслительный процесс

**Comment by artmotika** [comment]
Исправил замечания

**Comment by VladimirGl** [comment]
ок
